File: LeetCode/Simulation/JS/Leetcode1450-number-of-students-doing-homework-at-a-given-time.js
var busyStudent = function(startTime, endTime, queryTime) {
    let [ans,n] = [0,startTime.length];
    for(let i = 0;i < n;i++) {
        if(queryTime >= startTime[i] && queryTime <= endTime[i]) ans++;
    }
    return ans;
};

File: LeetCode/Simulation/Python/Leetcode1450-number-of-students-doing-homework-at-a-given-time.py
class Solution:
    def busyStudent(self, startTime: List[int], endTime: List[int],
                    queryTime: int) -> int:
        ans, n = 0, len(startTime)
        for i in range(n):
            if startTime[i] <= queryTime <= endTime[i]: ans += 1
        return ans
